File: demo/src/main/java/com/lcm/app/data/entity/db/Dog.java
package com.lcm.app.data.entity.db;

import io.realm.RealmObject;

/**
 * ****************************************************************
 * Author: LCM
 * Date: 2018/5/7 17:21
 * Desc:
 * *****************************************************************
 */
public class Dog extends RealmObject{

    private String name;

    private int age;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    @Override
    public String toString() {
        return "Dog{" +
                "name='" + name + '\'' +
                ", age=" + age +
                '}';
    }
}

File: demo/src/main/java/com/lcm/app/data/entity/db/Dog.kt
package com.lcm.app.data.entity.db

import io.realm.RealmObject
import io.realm.annotations.RealmClass

/**
 * ****************************************************************
 * Author: LCM
 * Date: 2018/5/7 15:31
 * Desc:
 * *****************************************************************
 */

open class Dog: RealmObject(){

    open var name:String?=null

    open var age:Int = 0


    override fun toString(): String {
        return "Dog(name=$name, age=$age)"
    }
}